### ISSUE TITLE
"Week by board" truncates time to the left of long board names

### ISSUE THREAD

**Issue by synrg** [zmandel/Plus-for-Trello#10]
Hi,

In Chromium 49 on Debian, whenever a board name is too long, the time is truncated to the left of the long board name when the page is at normal (100%) zoom level. For example, I have two boards, "Bluff Trail Maintenance and Upkeep" and "Membership/Volunteer Management" with this week's values 0.33 and 9.86 respectively. These show as "33 Bluff Trail Maintenance and Upkeep" and "86 Membership/Volunteer Management". My display is 1280x1024 and at work is 1440x900. The truncation is by the same amount on both displays when Chromium is maximized.

**Workaround:** if I zoom the page to 125%, I can see the full time by each, but then there is not enough screen width to see the full width of my boards without zooming, so I have to zoom back to 100% when I navigate to a board.

Here's what I see when drilling down into the individual Elements inside of the <svg> for the first of my two too-long board names in this report:

``` html
<text text-anchor="end" x="148" y="18.483333333333334" font-family="Arial" font-size="9"
 stroke="none" stroke-width="0" fill="#222222">0.33 Bluff Trail Maintenance and Upk</text>
```

Within the Developer Tools Elements browser, if I delete the last two characters "pk" in "and Upk" inside the &lt;text&gt; element, then the time will show within the 148 pixel space without truncation.

It is possible there is a related bug in the "Week by user", but our longest username is not long enough to trigger it yet.

I started reviewing plus.js and it looks like the issue occurs in handleRemapLabels/remapTextElements. When you alter the svg, you fail to truncate the elemSub.textContent on the right to accommodate for the mapRows[value]+" " added on the left.

Thanks,
Ben

**Comment by zmandel** [comment]
Indeed, we noticed this too when first implemented the chart. Back then we tried many alternatives but none were perfect. We tried the one you mention too, but at that point of execution we dont yet know the pixel space we have.

The current "remap" implementation is more of a hack arround the chart library's inability to customize labels in that way, and we will eventually replace both home charts with the new charting library that the board burndown uses (plottable.js), which does allow that type of customizations.

**Comment by synrg** [comment]
I'm looking forward to that. Thanks!
